openmetadata-ui/src/main/resources/ui/src/pages/Configuration/LoginPage/EditLoginConfigurationPage.tsx

```diff
@@ -0,0 +1,230 @@
+import { AxiosError } from 'axios';
+import React, { useEffect, useMemo, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useHistory } from 'react-router-dom';
+import ResizablePanels from '../../../components/common/ResizablePanels/ResizablePanels';
+import ServiceDocPanel from '../../../components/common/ServiceDocPanel/ServiceDocPanel';
+import TitleBreadcrumb from '../../../components/common/title-breadcrumb/title-breadcrumb.component';
+import Loader from '../../../components/Loader/Loader';
+import { VALIDATION_MESSAGES } from '../../../constants/constants';
+import {
+  GlobalSettingOptions,
+  GlobalSettingsMenuCategory,
+} from '../../../constants/GlobalSettings.constants';
+import {
+  CUSTOM_LOGIN_CONFIG_SERVICE_CATEGORY,
+  OPEN_METADATA,
+} from '../../../constants/service-guide.constant';
+import { ServiceCategory } from '../../../enums/service.enum';
+import { LoginConfiguration } from '../../../generated/configuration/loginConfiguration';
+import { Settings, SettingType } from '../../../generated/settings/settings';
+import { FieldProp, FieldTypes } from '../../../interface/FormUtils.interface';
+import {
+  getLoginConfig,
+  updateSettingsConfig,
+} from '../../../rest/settingConfigAPI';
+import { generateFormFields } from '../../../utils/formUtils';
+import { getSettingPath } from '../../../utils/RouterUtils';
+import { showErrorToast, showSuccessToast } from '../../../utils/ToastUtils';
+
+const EditLoginConfiguration = () => {
+  const { t } = useTranslation();
+  const history = useHistory();
+  const [form] = Form.useForm<LoginConfiguration>();
+  const [activeField, setActiveField] = useState<string>('');
+  const [loading, setLoading] = useState<boolean>(false);
+  const [updating, setUpdating] = useState<boolean>(false);
+
+  const fetchCustomLogoConfig = async () => {
+    try {
+      setLoading(true);
+
+      const data = await getLoginConfig();
+
+      form.setFieldsValue({ ...(data ?? {}) });
+    } catch (error) {
+      showErrorToast(error as AxiosError);
+    } finally {
+      setLoading(false);
+    }
+  };
+
+  const breadcrumb = useMemo(
+    () => [
+      {
+        name: t('label.setting-plural'),
+        url: getSettingPath(),
+      },
+      {
+        name: t('label.login-configuration'),
+        url: getSettingPath(
+          GlobalSettingsMenuCategory.OPEN_METADATA,
+          GlobalSettingOptions.LOGIN_CONFIGURATION
+        ),
+      },
+      {
+        name: t('label.edit-entity', {
+          entity: t('label.login-configuration'),
+        }),
+        url: '',
+      },
+    ],
+    []
+  );
+
+  const formFields: FieldProp[] = [
+    {
+      name: 'maxLoginFailAttempts',
+      label: t('label.max-login-fail-attampt-plural'),
+      type: FieldTypes.NUMBER,
+      required: false,
+      id: 'root/maxLoginFailAttempts',
+      props: {
+        'data-testid': 'maxLoginFailAttempts',
+        size: 'default',
+        style: { width: '100%' },
```

Comment:
Do we need this style prop?

Reply:
Yes else it will be of just 90px which wasn't good